doc/content/lorawan/rssi-and-snr/_index.md

```diff
@@ -0,0 +1,54 @@
+---
+title: "RSSI and SNR"
+section: Additional Information
+description: ""
+weight:
+---
+
+In wireless communication, a receiver needs a good signal strength and a signal-to-noise ratio to separate the original information or signal from the modulated carrier.
+
+## RSSI
+
+**RSSI (Received Signal Strength Indicator)** is a measurement that helps you to determine if the receiver is receiving enough signal (signal strength) to get a good wireless connection from the transmitter. Since LoRaWAN supports bi-directional communication, RSSI is an important measurement for both gateways and end devices. RSSI is measured in dBm and it is a negative value. An RSSI value that is closer to zero usually means a better signal strength.
+
+Apart from the output power of the transmitter, the following factors mainly influence the RSSI. 
+
+- Path loss
+- Antenna gain
+- Cable/connector loss
+
+## SNR
+
+**Signal-to-Noise Ratio**, often written as SNR or S/N, is the difference between the power of the received signal and the noise floor. SNR can be used to determine the quality of the received signal.
+
+SNR is calculated using the following formula and is typically expressed in decibels (dB).
+
+**SNR (dB) = received signal strength - noise floor**
+
+If the RSSI is above the noise floor the receiver can easily demodulate the signal.
+
+Here is a good example of positive SNR:
+
+{{< figure src="snr-greater-than-zero.png" alt="SNR is greater than zero" >}}
+
+_Image captured from the [YouTube video](https://www.youtube.com/watch?v=QFapH4oADnU) by Richard Wenner. The x-axis represents the power level in dBm and the y-axis represents the time._
+
+By looking at the above graph you can see that the RSSI is about -65 dBm and the noise floor is about -90 dBm. By using these values you can calculate the SNR as follows.
+
+SNR (dB) = received signal strength - noise floor
+
+SNR (dB) = -65 -(-90) = 25 dB (positive SNR)
+
+If the RSSI is below the noise floor, it is impossible to demodulate the signal. However, LoRa operates below the noise floor. Therefore, the receiver can demodulate the signal if the RSSI is below the noise floor. 
```

Comment:
I would add a short explanation of how LoRa manages to operate below the noise floor, or at least add a link to some literature explaining it